deployments/k8s/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for asyncapi-event-gateway.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: asyncapi/server-api
+  pullPolicy: Always
+  tag: "latest"
+
+imagePullSecrets: []
+
+replicaCount: 2
+
+# Server API opened ports.
+ports:
+  # Port where the Server API will be available.
+  http: 5000
+
+# Env vars. Needed for configuring the app.
+env: {}
```

Comment:
The fact there are no env vars seems like a smell to me. Something we have to work on soon so the whole app config can be set via environment variables. An interesting approach to make saas is https://12factor.net, where their  [config](https://12factor.net) section advocates for environment variables (which I'm totally in as well)

Reply:
I know that and I'm all for environment variables too :) Don't worry, we will do it but not in this PR. I can remove the `envs` field but I don't know if it is that important.

Reply:
No,it is ok to leave it here.

Reply:
Issue for that is created https://github.com/asyncapi/server-api/issues/39 :)